guides/common/modules/proc_configuring-pull-transport.adoc

```diff
@@ -0,0 +1,31 @@
+[id="configuring-pull-provider_{context}"]
+= Configuring Remote Execution for Pull Client
+
+By default, Remote Execution uses SSH as the transport mechanism for the Script provider.
+However, Remote Execution also offers a pull-based transport option, which communicates with hosts via MQTT.
+This is a good option if you are migrating from `katello-agent` which also uses pull-based communication.
+
+If you wish to use Remote Execution with the pull-based transport, run the following `{foreman-installer}` command on each relevant {SmartProxy}:
+
+[options="nowrap" subs="quotes,attributes"]
+----
+
+ # {installer-scenario-smartproxy} \
+  --foreman-proxy-plugin-remote-execution-script-mode pull-mqtt
+
+----
+
+The mode is configured per {SmartProxy}.
+Some {SmartProxies} may be configured to use `pull-mqtt` mode while others use `ssh`.
+If this is the case, it is possible that one remote job on a given host will use the pull client, and the next job on the same host will use SSH.
+If you wish to avoid this scenario, configure all {SmartProxies} to use the same mode.
+
+In `pull-mqtt` mode, hosts subscribe to the {SmartProxy} they are registered through for job notifications.
```

Comment:
This paragraph sounds like a different procedure.

Reply:
See [above](https://github.com/theforeman/foreman-documentation/pull/1548/files#r961579453) for some context. I agree it does kind of feel like a different procedure, but it should be done at the same time you configure your {SmartProxies} for pull mode. Given that, what's the best place for this text?

Reply:
You can potentially split the procedure into two. But IMHO this can be done at a later stage.